core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -545,4 +545,4 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       this.changeDetectorRef.detectChanges();
     }
   }
-}
+}
```

Comment:
I am not sure why this is being included as a change ? Also I don't know what's the change here.